.github/workflows/build-images-ci.yaml

```diff
@@ -244,6 +249,41 @@ jobs:
             NOSTRIP=1
             OPERATOR_VARIANT=${{ matrix.name }}
 
+      - name: Install Bom and generate SBOM
+        shell: bash
+        run: |
+          curl -sSL --remote-name-all https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz
+          sudo tar xzvfC bom-linux-amd64.tar.gz /usr/local/bin
+          rm bom-linux-amd64.tar.gz
+          bom generate -o sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --dirs=.
+          bom generate -o sbom_image_ci_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}
+          bom generate -o sbom_image_ci_race_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
+          bom generate -o sbom_image_ci_unstripped_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-unstripped
+          
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@ced07f21fb1da67979f539bbc6304c16c0677e76
+      
+      - name: Attach SBOM to container images
+        env:
+          COSIGN_EXPERIMENTAL: "true"
+        run: |
+          cosign attest --type spdx -predicate sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}
+          cosign attest --type spdx -predicate sbom_image_ci_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}
+          cosign attest --type spdx -predicate sbom_image_ci_race_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
+          cosign attest --type spdx -predicate sbom_image_ci_unstripped_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-unstripped
+
+      - name: Verify SBOM attestation
```

Comment:
I'm not sure that you need to do verification here because you are the provider of the software and your responsibility is to sign and distribute it, so later on, the customers of your software should verify your software before deciding whether they use it (optionally).

Reply:
You're right, this is not necessary. I added the ` - name: Verify SBOM attestation` step to test if verifying SBOM  attestation is working correctly and the `- name: Attach SBOM to container images` worked correctly. But ` - name: Verify SBOM attestation` step should not keep running for more than 5.5 hours and ultimately fail with an error if everything is working as expected.